Source/WebCore/css/CSSPageRule.cpp

```diff
@@ -65,8 +65,7 @@ String CSSPageRule::selectorText() const
 void CSSPageRule::setSelectorText(const String& selectorText)
 {
     CSSParser parser(parserContext());
-    auto* sheet = parentStyleSheet();
-    auto selectorList = parser.parseSelectorList(selectorText, sheet ? &sheet->contents() : nullptr);
+    auto selectorList = parser.parsePageSelector(selectorText);
```

Comment:
Why are we not passing the `StyleSheetContents` anymore ?? 

`static CSSSelectorList parsePageSelector(CSSParserTokenRange, StyleSheetContents*);`

Reply:
It is for detached @page and it does not require stylesheet linkage.

Reply:
If we passed it before, was it **harmless** or **wrong**? I presume it had to have been **wrong**, and so we need to demonstrate the wrong behavior with a test.